app/views/transactions_mailer/transaction_notification.html.erb

```diff
@@ -1,33 +1,8 @@
-<h1>Dear <%= @account.name%>, transaction added!</h1>
+<h1>New transaction was added in <%= @account.name %>!</h1>
 <h3>About transation:</h3>
-<p>From:</p>
-<%= @transaction.from_account.name %>
-<p>To:</p>
-<%= @transaction.to_account.name %>
-<p>Amount:</p>
-<%= @transaction.amount %>
-<p>Description:</p>
-<%= @transaction.description %>
+<p>From: <%= @transaction.from_account.name %> </p>
+<p>To: <%= @transaction.to_account.name %> </p>
+<p>Amount: <%= @transaction.signed_amount(@account) %> </p>
+<p>Description: <%= @transaction.description %> </p>
 <br>
-<p>Click to link for approve transaction </p><%= link_to 'Approve', '#' %>
-<h3> Last 5 transactions: </h3>
```

Comment:
Let's keep thee last 5 transactions. I see no harm in them. But it's better to move this table lower.